bloom-housing/bloom
# feat: frontend for allowing marking status for application flagged sets

# Pull Request Template

## Issue Overview

This PR addresses #2832 


## Description

Sets up being able to mark individual applications in a flagged set as resolved.

Offline design changes:
On the individual flag set page, the id is a pinned left column, and the status is a pinned right column.

## How Can This Be Tested/Reviewed?

Check out the issue and the designs. 

Visit a closed listing. Go to the applications tab and then the pending review tab. Click into an application flagged set. This page and the resolved page are the two main pieces of the ticket.

On the pending page:
* All applications should have a default status of `pending`.
* You can select any number of applications and when you hit the save button. If you save as "pending", the set stays as pending and the selected applications move to `valid (pending)`. If you save as `resolved` the set moves to `resolved` and the selected applications move to `valid` and the unselected applications move to `duplicate`.
* When you move a set from pending to resolved, it should appear in the appropriate side nav tab.
* The leftmost and rightmost columns should be pinned
* Check out mobile on the set page (it was lots of tinkering to get the side nav to be accessible, recognizing the pinned columns makes the table challenging, but living with that for now)


## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
